src/main/scala/com/databricks/spark/sql/perf/tpcds/Tables.scala

```diff
@@ -53,43 +65,71 @@ class Tables(sqlContext: SQLContext, dsdgenDir: String, scaleFactor: Int) extend
 
           // Note: RNGSEED is the RNG seed used by the data generator. Right now, it is fixed to 100.
           val parallel = if (partitions > 1) s"-parallel $partitions -child $i" else ""
+          val prefix = if (parentTable.isDefined) "-WRITE_PREFIX Y" else ""
+          val table = if (parentTable.isDefined) parentTable.get else name
           val commands = Seq(
             "bash", "-c",
-            s"cd $localToolsDir && ./dsdgen -table $name -filter Y -scale $scaleFactor -RNGSEED 100 $parallel")
+            s"cd $localToolsDir && ./dsdgen -table $table -filter Y -scale $scaleFactor -RNGSEED 100 $parallel $prefix")
           println(commands)
           commands.lines
         }
       }
 
+      if (parentTable.isDefined) {
```

Comment:
Davies is working on a better way to do this. I'd hold off looking at this any more. I think we can just make all the changes in dsdgen and leave this as is.